internal/surgeon/surgeon.go

```diff
@@ -16,19 +17,105 @@ func CopyPage(path string, srcPage common.Pgid, target common.Pgid) error {
 }
 
 func ClearPage(path string, pgId common.Pgid) error {
+	return ClearPageElements(path, pgId, 0, -1)
+}
+
+func ClearPageElements(path string, pgId common.Pgid, start, end int) error {
```

Comment:
Please document that:
  - it supports both: branch & leaf pages. 
  - (how it behaves in respect to freelist -> please see comment below)

Reply:
Resolved.